roles/server/tasks/symfony.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: copy parameters.yml for symfony project
+  template:
+    src=symfony_parameters.yml
+    dest='{{ home }}/site/app/config/parameters.yml'
+
+- name: check vendor/ for symfony project
+  stat: path='{{ home }}/site/vendor/'
+  register: vendor
+
+- name: composer install for symfony project
+  composer:
+    working_dir='{{ home }}/site'
+    prefer_dist=yes
+    no_dev=no
+  ignore_errors: yes # warnings about an outdated composer.lock file might be sent to stderr, which makes Ansible think the task failed... it didn't, so we'll ignore errors
```

Comment:
Does it not look at the return code of the command rather than what's sent to `stderr`?